src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -8435,7 +8447,7 @@ paths:
         '404':
           description: |
             **actionDefinitionId** or **targetId** does not exist
-  '/nodes/{nodeId}/rules':
+  '/nodes/{nodeId}/rulesets/{rulesetId}/rules':
```

Comment:
If this is -default- (or similar) then should be stated in each method description (alongside folder "nodeId").